lbz/response.py

```diff
@@ -17,21 +21,37 @@ def __init__(
         base64_encoded: bool = False,
     ):
         self.body = body
-        self.is_json = isinstance(body, dict)
-        self.headers = headers if headers is not None else self.get_content_header()
+        self.headers = headers if headers is not None else self._get_content_header()
         self.status_code = status_code
+        # TODO: handle bae64 encoded responses appropriately
         self.is_base64 = base64_encoded
+        self._json: dict | None = None
+        if isinstance(body, dict):
+            self._json = body
```

Comment:
I would declare it right under the `body`, as these two attributes are very connected:
```suggestion
```